job-service-container/pom.xml

```diff
@@ -161,7 +162,7 @@
                 <webservice.https.adminport>25083</webservice.https.adminport>
                 <rabbitmq.node.port>5672</rabbitmq.node.port>
                 <rabbitmq.ctrl.port>15672</rabbitmq.ctrl.port>
-                <postgres.db.port>5432</postgres.db.port>
+                <postgres.db.port>${postgres.db.port}</postgres.db.port>
```

Comment:
Why is this changing?

Reply:
I was using this change to connect to a fixed port in my local while development. This was pushed by mistake. Reverted.